lib/hawkular/inventory/inventory_api.rb

```diff
@@ -429,6 +429,17 @@ def create_metric_type(feed_id, metric_type_id, type = 'GAUGE', unit = 'NONE', c
       MetricType.new(new_mt)
     end
 
+    # List operation types for given resource type
+    # @param [String] resource_type_path canonical path of the resource type entity
+    # @return [Array<String>] List of operation type ids
+    def list_operation_types(resource_type_path)
+      parsed_path = CanonicalPath.parse(resource_type_path)
+      feed_id = parsed_path.feed_id
+      resource_type_id = parsed_path.resource_type_id
+      ret = http_get("/feeds/#{feed_id}/resourceTypes/#{resource_type_id}/operationTypes")
+      ret.map { |ot| ot['id'] }
```

Comment:
why is it beneficial to the user to get operation type ids? what can a user do with an operation id (instead of the operation name/description)? also, the method name is misleading - the client names it "list_operation_types" while it returns ids.